ui/pages/institutional/pin-mmi-billboard/pin-mmi-billboard.js

```diff
@@ -0,0 +1,597 @@
Yzcb/hhBSRJThDpEFKUuPfbyEZ5blJgRLYAIwILAQZBbAyawJajAR8O2cb/gChBk6MxITFWP5hSI4ab3Ji1njkr1qPBvp/UbQPVMvQRvuhQVS3Z1KRWmCJ4r1WgClSBKlAFXq7AfoNHKqwgufhDuCE/EBc4RqAhKqjEgntsoeEXFRlMz1+kIVjYuNb/4UhT8zUlKQ2JfFTFiTFcpwImuLmgj6Eq3syzGa1zRYpB2qbiFgB5JUemvUHMAmFM3Xq8QcJTRLorMEWVXqtAFagCVaAKvFgBgUbe4s+DK8lDzEvnmIRAZweNcARiCFH4Gmja21oBE043ucQjylaBHbozjMj1ZeVWmUCnT8dlIHbMNhMcgdAOMYdWJwCtRtz4eXTH9CRO2xfEecUQmYzQZ7fkjn5tVYEqUAWqQBV4tQIBHqNC3vfTnfvAyEUUD7o4IAL88CAuPuuT6bS1D3kvUGH16TiVtAnmr0KfM0xAHsHK44C4jyH9ZkJYhv9ypJBzJybEnTRQFWpAj7HytDmh6um7FabfFG9fFagCVaAKVIGXKSAs+J1IAkm5JjVUWVJBSV+u6DcjqcumccEfoWnPMRxxmktDrFIFmmAlpBUnvp64AjPZTnSlzGtTccp8Xh2XpgiCCHn2FeTDLdYLAEVdEw5ijPFfYkJXgcnC9FIFqkAVqAJV4N9QIASBq9qCIt0ThM75blVbyA887S0JWElKhSf2NBTwwN14CBQZoEZHFmwGS6b72TjQJp8BIvWD6uQskUw+BB1ZyxpCGmpMHssaHdp6evZ9wDFfwnvBEwcLTH+Rrt1VoApUgSpQBV6vQCACxHHaSiusEbDg85LAJRlgRUaww+rV5h5QiKFLH7kzmuXTZqjI5NNwgRC4HXp5tB+lJ8SAMK4f+L8KQaQxTlHVyBUsxpW6kopaNLXywqIxn/ksP42lOeskB58FpuvF6E0VqAJVoApUgdcq8CfRkBHYjcrNYYDASK6syvBBjdiZOrBxTv9oLSBj/lZBCtUburETzkm1KWPQdaAntBWxgy8L7JwOHrappessE4tODsBLGStzjK1DRXLrfoCRwUs9PjwOJ4jf1aVlhf67JUcZ+q8KVIEqUAWqwNsVwFu8KWNS0dYUIeg5lPt1TX0loCR0yv/bOpalasWKUPiEQGLw2G3HhDWsYk2MdwNjtEsTDo4X5ULIw5gffaDVwpvEzT4GeGwztwdQ2fomOS9IDN7a6xmmI2RbVaAKVIEqUAX+BQUIBkzkVGuyxeXiyYCOp/ESANnQERAaXVwYAmgAUQI3AjJVgNB3/cSpO7f9sy92rRigGlNXNUtzYQcxk8mIPCdzzGAcZsjLZ+LYNuno4NPEtapj/7lD7F0VqAJVoApUgSrwTgUEDIIDvfkLONROsWdyu7gmqxY0+GwP5ps71LABzET/AJIdqDrEUs64Yp/BaeZjdLmbyezf22YBIQSkoH+uHb2Mn/AuQJpqlNwpDK8ZImMiCgExMa5NVP2U3PX69KYKVIEqUAWqwJsVEE4EZf6eCaCFEOMp4YYnaBCI/MTvsbYgZzVneBogE/8prgVXZ9IVBzfasIbx8aJjT2QXb8lhLQ2es0eIG7/AoCuveSLTYjOuNdDZV84t0bRtixJPhj3DNC9aG1WgClSBKlAF3qxAqjp4xxc0gQfwQy5gY840ezst1RSt0CoSgxaKTA52aILGYNSfipODvc7LY2V/Uu4ayhkhBSp+UfypROWqj82tyhMojynLL0CJwOeKUupOlmCICbMJiHJmqRw7tuDcn08NJtwCU5TotQpUgSpQBarAqxXIPpXO9SCVqRyRElSZESwoUbRPVQZoYbwglxhE2KcHQpqfZjErVdZsKjs0OpjiUcRiKJme3dDYmbLnJp8dm0IF21w/C6CuftzYJJZwHSBuaZT5jPPz8+MbnxrEjyb3sQIRqNcqUAWqQBWoAu9WINtsqNogE9FEoALXtAMPmAVAmF8TCy4EKX4SjbOCUoKMwx8y+vPz8c1FASvBRngGMKXKTRTOSO5zNaQsb4jZYWlSphh2AmoBMtwLhRCa/OSeBmIvfBnXnHs+HZe1IC34b4UpQvVaBapAFagCVeDlCggsAAmgivwpqYyRC8IrgIYvb15huvszDNBAGzPCKbo/MAKaAFygH9BC0BhQsaAhHhv28KX2ARRaMt3p+VGYqDWwr9Rw0RpFRtP8p7NMNO6gFT8pkLnEMYb5gE13JMxDlvTM3ApMUa3XKlAFqkAVqAKvV8BAYcwB6gwEMDdXn8AOIA8Qg7emuGTgKBByylJkka0P4Aj3fq7SwMd2iDbv79UDVnFq2LK5AbANVlrzNfkwdCYAHDpAd7kP5tm9wnHcBi5lKh2kyfSIzLDos0/63i9921WgClSBKlAFXqtAKjR+f2ceBzgCUnzvJ5BkC2sAw9twWMh6URjHW3a/CoM5dsgm17pvDLtz33OKkCd2T/yCOsRHdBlmU0DIiXkN16hyNGzkpLXaSdiQ7+SSffj3wypT2G3MT2Nk3wAAAVFJREFUOo1Um1phyivVaxWoAlWgClSBFysQAAIAqH2uSAu8QjwYUAB4+GxRqi3rPBMmj02vveQxIcmmqjCBJ84bsrlW/XEjUHK1SwvFKNzmE+IgDszDNXlchhAEQCrUw1uvxUSO+Wqg0lT5nTwxBeP0cyATJgpMl+K9qQJVoApUgSrwVgWu+omTCBCJGDAjvKAJxJ2DGZigLg3bpKDmHuI8Vns+VaEByRg2SCiwsPs4NiOOL1CGepCgaJZx+TedBpYSBxc7NoWrc0sBH1aXDEkCIMESYYsx2f1uW4XAmXzZSYFpXq82qkAVqAJVoAr8QwoElMQsSCznfMAI+VEbUKDqDSCBzx8KKLk0Q5jxooMQgisAxnxCjgDiT6nR+JlNmNmFpASxroIVdLg6ZrrjWgKUoSo5LLhjjF/xLaMEpyTsnTta5/rE5qvn0SW6coDd/f8F1TN8RwHfQboAAAAASUVORK5CYII="
```

Comment:
what is this? xlinkHref? Is this a data encoded image that we are embedding in an SVG? This feels like a pretty hefty svg file

Reply:
Yes,  it's almost the same as the default MM one in ui/pages/onboarding-flow/pin-extension/pin-billboard.js just the colours and a string text for the app name that change.